From 119074a848ce048bafc2c53846a7f9e97920e0b7 Mon Sep 17 00:00:00 2001
From: Joseph Liu <kwliu@nuvoton.com>
Date: Fri, 20 Dec 2019 15:21:23 +0800
Subject: [PATCH] dts: runbmc-olympus: enable fan filter for fan0-2

There is a noise issue on fan tach. channel that
we should enalbe noise filter for fan0-2.

Signed-off-by: Joseph Liu <kwliu@nuvoton.com>
---
 arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts b/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts
index e328a8bd6aaa..fceb13b956d4 100644
--- a/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts
+++ b/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts
@@ -985,16 +985,19 @@
 					reg = <0x00>;
 					fan-tach-ch = /bits/ 8 <0x00 0x01>;
 					cooling-levels = <127 255>;
+					fan_filter_en;
 				};
 				fan@1 {
 					reg = <0x01>;
 					fan-tach-ch = /bits/ 8 <0x02 0x03>;
 					cooling-levels = /bits/ 8 <127 255>;
+					fan_filter_en;
 				};
 				fan@2 {
 					reg = <0x02>;
 					fan-tach-ch = /bits/ 8 <0x04 0x05>;
 					cooling-levels = /bits/ 8 <127 255>;
+					fan_filter_en;
 				};
 				fan@3 {
 					reg = <0x03>;
-- 
2.17.1

